_posts/2020-11-25-cockpit-233.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 233
+author: lis
+date: 2020-11-25
+tags: cockpit linux
+slug: cockpit-233
+category: release
+summary: Cockpit with various bug fixes and performance improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 233.
+
+### Non-admin users no longer see Cockpit in motd
+
+The motd message suggesting how Cockpit can be enabled or used is now
+shown only to admin users.  This depends on PAM 1.5.0 (which is not yet
+released) but the needed patches have been backported to Fedora 33 and
+may land in other distributions in the future.
+
+### Stop building and shipping jQuery
+
+The long-deprecated `../base1/jquery.js` has been removed from Cockpit.
+All known external users have been ported away.
```

Comment:
Does the jQuery removal impact developers? 

If not, I agree with @Gundersanne.

If so, it would be a developer-related note and we should probably show it as such:

```suggestion
## Developers: jQuery removal

Third party developers: Take note! [As previously announced](https://cockpit-project.org/blog/cockpit-163.html), we have finally removed the long-deprcated `../base1/jquery.js` from Cockpit. The bundled version is no longer built or shipped. If you are using jQuery in your application, be sure to ship a version yourself.

All known external users have been ported away.

[Read more details in the original announcement](https://lists.fedorahosted.org/archives/list/cockpit-devel@lists.fedorahosted.org/thread/TDBKI4372I4ZPGQFCZB377DX2QG2YDDY/).
```